commit e80a4b87cfa98708724942794ca6f051974bb5c0
Author: Eike Hein <hein@kde.org>
Date:   Tue Jan 3 16:30:54 2017 +0900

    Don't clear hover state while a child dialog is open.
    
    Otherwise it won't stay open.
    
    Fixes regression from 7585f295363532eff51f7e11ef6b62925117aba4.
    
    The bug number referenced in the original change is invalid, but
    I don't think this should invalidate the intent of that fix. Kai
    please let me know the correct bug number so I can double-check.
    
    BUG:374291
    CCMAIL:kde@privat.broulik.de

diff --git a/containments/desktop/package/contents/ui/FolderView.qml b/containments/desktop/package/contents/ui/FolderView.qml
index 1fd8dbb..8936532 100644
--- a/containments/desktop/package/contents/ui/FolderView.qml
+++ b/containments/desktop/package/contents/ui/FolderView.qml
@@ -343,7 +343,10 @@ Item {
         onContainsMouseChanged: {
             if (!containsMouse && !main.rubberBand) {
                 clearPressState();
-                gridView.hoveredItem = null;
+
+                if (gridView.hoveredItem && !gridView.hoveredItem.popupDialog) {
+                    gridView.hoveredItem = null;
+                }
             }
         }
 
